homework/Vasili_Zasinets/src/test/java/com/tutrit/java/quickstart/homework/ball/MagicTest.java

```diff
@@ -0,0 +1,34 @@
+package com.tutrit.java.quickstart.homework.ball;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MagicTest {
+    Magic magic;
+    Logger log = LoggerFactory.getLogger("Ball ");
+
+
+    @Before
+    public void setUp() {
+        this.magic = new Magic();
+    }
+
+    @Test
+    public void addPrediction() {
```

Comment:
These test is true always.
I thing it need rewrite.
You can use ball.size()

Reply:
fixed